### PR TITLE
[Feat] Webhook HA

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr makes it possible to set multiple replicas for the liqo webhook deployment

### CODE REVIEW

**Review by fra98 — APPROVED** [review]

File: pkg/leaderelection/labeler.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			maxRetries--
+			return true
+		}, func() error {
+			if err := handleLeaderLabelWithClient(ctx, cl, info); err != nil {
+				klog.Error(err, "retrying...")
+				return err
+			}
+			return nil
+		}); err != nil {
+			klog.Error(err)
```
Comment (claudiolor) [inline pkg/leaderelection/labeler.go:54]: Wondering whether it is worth panicking when the leader is not able to add itself the label. As in that case we will be in a "gray" condition, where the leader is running but it actualy doesn't handle the incoming requests

**Pull request merged**